guide/payments/contacts.md

```diff
@@ -0,0 +1,229 @@
+    - file: activity-annotated
+      alt: Activity screen showing transactions with assigned contacts
+      caption: Assigning contacts to payments makes them much easier to understand and work with.
+    - file: activity-transaction
+      alt: Transaction details
+      caption: Transaction detail screens should have convenient options to assign a contact.
+    - file: sending-pick-contact
+      alt: Screen with a list of contacts to choose from
+      caption: The user picks a contact in a slide-up modal.
+    - file: activity-transaction-annotated
+      alt: Transaction details with assigned contact
+      caption: A payment with an assigned contact.
+
+---
+
+{% include picture.html
+    image = "/assets/images/guide/payments/contacts/contacts-header.jpg"
+    retina = "/assets/images/guide/payments/contacts/contacts-header@2x.jpg"
+    mobile = "/assets/images/guide/payments/contacts/contacts-header-mobile.jpg"
+    mobileRetina = "/assets/images/guide/payments/contacts/contacts-header-mobile@2x.jpg"
+    alt-text = "Illustration of large contact cards"
+    width = 1600
+    height = 400
+    layout = "full-width"
+%}
+
+
+# Contacts
+{:.no_toc}
+
+<!--
+
+
+-->
+
+Whether we’re sending emails, physical mail, or following someone on social media, we primarily think in terms of names and faces, and not the respective address or user ID.
+
+Invoices, node IDs and other transaction endpoints in bitcoin and Lightning are highly unintuitive. Abstracting them via a contact list can create a much smoother user experience. There are many [payment request formats]({{ '/guide/payments/request/payment-request-formats/' | relative_url }}), each with unique properties and varying levels of maturity and adoption, requiring unique design solutions. This page uses the more approachable term "address", along with various UI techniques, to abstract these complexities for users.
+
+Let's go over common user interactions around contacts. This will illustrate how such a feature could work, and helps explain the underlying design problems and decisions.
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+### Adding a contact
+
+The most basic interaction is that a user manually adds a contact by entering their name and an address. This is likely not the most common user flow, as most contacts will be added to incoming payment requests, as illustrated further below. But wallets should generally support manual options, as users may not be able to avoid them.
+
+{% include image-gallery.html pages = page.imagesAddContact %}
+
+### Supporting various payment request formats
+
+Based on your use case, your application may choose to only support certain payment request formats. It should still recognize all formats, and provide the user with appropriate feedback. You may also choose to add warnings before permitting use of certain formats if there are privacy and security risks the user should be aware of.
```

Comment:
```suggestion
Based on your use case, your application may choose to only support certain payment request formats. It should still recognize all formats, and provide the user with appropriate feedback. You may also choose to add warnings before permitting use of certain formats if there are privacy and security risks the user should be aware of.

Only [reusable payment requests]({{ '/guide/payments/request/#reusable-payment-requests' | relative_url }}) should be possible to be imported by users. 
```

Reply:
I assume by `import`, you mean `save to contact for future use`. That sentence goes counter to the first screen in the gallery, which shows a single-use on-chain address.

Reply:
> I assume by import, you mean save to contact for future use.

Yep!

> That sentence goes counter to the first screen in the gallery, which shows a single-use on-chain address.

Personally I'd just remove the options around saving an on-chain address (and xpubs) to a contact and stick with reusable payment requests only. On-chain payments should be more intentional so the generating of a new address each time isn't a big burden

Reply:
I want to keep them, since this section is all about how to highlight various properties of different formats in the UI, so users can work with them without having to understand anything technical. The text is clear in that designers should think deeply about this and decide what to include based on their use cases.